src/server/services/kvk.ts

```diff
@@ -14,12 +14,12 @@ export interface Onderneming {
   overigeActiviteiten: string[] | null;
   datumAanvang: string;
   datumEinde: string | null;
+  kvkNummer?: string;
 }
 
 export interface Rechtspersoon {
   rsin?: string;
-  bsn?: string;
-  kvkNummer: string;
+  kvkNummer?: string;
```

Comment:
Deze `?` mag weg

Reply:
Maar in geval van zzp wordt deze toch naar de onderneming gestuurd? Of moet het string | null zijn?

Reply:
Ja, maar een ZZP heeft nooit een rechtspersoon. Dus als er een rechtsperoon aanwezig is, is er een kvk aanwezig